loculus_values/values.yaml

```diff
@@ -1332,17 +1332,12 @@ organisms:
           log_level: DEBUG
           nextclade_dataset_name: nextstrain/cchfv/linked
           nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/cornelius-cchfv/data_output
-          nucleotideSequences: [L, M, S]
           genes: [RdRp, GPC, NP]
     ingest:
       <<: *ingest
       configFile:
         <<: *ingestConfigFile
         taxon_id: 3052518
-        nucleotide_sequences:
```

Comment:
Nice, how is this now unnecessary? :)

Reply:
I now create this from the values.yaml in the template: https://github.com/loculus-project/loculus/pull/3075/files#diff-527ac956ba2c93519e4a76a03d7b123ba2dbd479147ce94582a4f07d3d07bad1